src/stylesheets/components/_hints.scss

```diff
@@ -0,0 +1,8 @@
+.url {
+	padding-right: $base-spacing-unit;
+	width: 80%;
```

Comment:
What if the hint is long - e.g. 25% of space? Might be worth converting to flex.

Reply:
Hints were not part of this change just a move around of the existing styles done previously. Won't be looking into as part of this commit.

Reply:
👍